### PR TITLE
Embedding avatar in guest invite emails

### PR DESCRIPTION
**Pull request by jespino**
#### Summary
Avatar images were referenced before instead of using the avatar image embeded
in the email. I changed that to be sure that the image is properly loaded for
somebody without any permission to see the image.

#### Ticket Link
[MM-17197](https://mattermost.atlassian.net/browse/MM-17197)

### CODE REVIEW

**Comment by srkgupta** [comment]
Hi @jespino 
Can you also please fix the alignment issues seen on the Gmail client as originally reported in the ticket. The name and the message is center aligned.

**Comment by jespino** [comment]
@srkgupta I was checking the correct behavior in inbucket visualization and in thunderbird, but not in gmail, I going to test it.

**Comment by jespino** [comment]
@srkgupta Done! can you test it in the spinmint if you want :)

File: services/mailservice/mail.go [hunk 1]
```diff
@@ -208,11 +208,11 @@ func SendMailUsingConfig(to, subject, htmlBody string, config *model.Config, ena
 	fromMail := mail.Address{Name: *config.EmailSettings.FeedbackName, Address: *config.EmailSettings.FeedbackEmail}
 	replyTo := mail.Address{Name: *config.EmailSettings.FeedbackName, Address: *config.EmailSettings.ReplyToAddress}
 
-	return SendMailUsingConfigAdvanced(to, to, fromMail, replyTo, subject, htmlBody, nil, nil, config, enableComplianceFeatures)
+	return SendMailUsingConfigAdvanced(to, to, fromMail, replyTo, subject, htmlBody, nil, nil, nil, config, enableComplianceFeatures)
 }
 
 // allows for sending an email with attachments and differing MIME/SMTP recipients
-func SendMailUsingConfigAdvanced(mimeTo, smtpTo string, from, replyTo mail.Address, subject, htmlBody string, attachments []*model.FileInfo, mimeHeaders map[string]string, config *model.Config, enableComplianceFeatures bool) *model.AppError {
+func SendMailUsingConfigAdvanced(mimeTo, smtpTo string, from, replyTo mail.Address, subject, htmlBody string, attachments []*model.FileInfo, embeddedFiles map[string]io.Reader, mimeHeaders map[string]string, config *model.Config, enableComplianceFeatures bool) *model.AppError {
```
Comment (ali-farooq0) [inline services/mailservice/mail.go]: oh boy 11 arguments :P i think it's time we convert this into a struct? :P

Comment (jespino) [reply services/mailservice/mail.go]: I think is a good idea to rethink that, but probably not for a pr to be cherry picked to 5.16, I going to create a ticket for that to do the change in master.

Comment (ali-farooq0) [reply services/mailservice/mail.go]: agreed.

**Pull request merged**